test/utils/pickers-utils.tsx

```diff
@@ -77,14 +96,21 @@ export function createPickerRenderer({
 }: CreatePickerRendererOptions = {}) {
   const { clock, render: clientRender } = createRenderer(createRendererOptions);
 
-  let adapterLocale = (adapterName ?? adapterToUse.lib) === 'date-fns' ? locale : locale?.code;
+  let adapterLocale = [
```

Comment:
```suggestion
  let adapterLocale = (adapterName ?? adapterToUse.lib).includes('date-fns')
```

Just a suggestion

Reply:
It took me some time to figure out I need to include `js-joda` here so I prefer to keep the comment for latter or for the other dev who will add it